baselines/driving_smarts/benchmark.py

```diff
@@ -0,0 +1,140 @@
+# smarts/benchmark/driving_smarts_competition_benchmark.py
+import argparse
+import logging
+from functools import partial
+
+import gymnasium as gym
+import ray
+
+from smarts.benchmark import auto_install
+from smarts.benchmark.driving_smarts import DEFAULT_CONFIG, load_config
+from smarts.env.gymnasium.wrappers.episode_limit import EpisodeLimit
+from smarts.env.gymnasium.wrappers.metrics import CompetitionMetrics
+from smarts.zoo import registry as agent_registry
+
+LOG_WORKERS = False
+
+
+@ray.remote(num_returns=1)
+def _eval_worker(name, env_config, episodes, agent_config):
+    import warnings
+
+    warnings.filterwarnings("ignore")
+    env = gym.make(
+        env_config["env"],
+        scenario=env_config["scenario"],
+        **env_config["shared_params"],
+    )
+    env = EpisodeLimit(env, episodes)
+    env = CompetitionMetrics(env)
+    agent = agent_registry.make_agent(
+        locator=agent_config["locator"],
+        **agent_config["params"],
+    )
+
+    observation, info = env.reset()
+    try:
+        while not info.get("reached_episode_limit"):
+            try:
+                # action: [global x-coordinate, global y-coordinate]
+                action = {
+                    agent_id: agent.act(obs) for agent_id, obs in observation.items()
+                }
+                # assert env.action_space.contains(action)
+            except Exception:
+                logging.error("Policy robustness failed.")
+                # env.mark_policy_failure()
+                terminated, truncated = True, True
```

Comment:
If an episode ends due to error, then I guess:
`terminated = False`
`truncated = True`